assets/src/edit-story/app/rightClickMenu/constants.js

```diff
@@ -47,6 +50,26 @@ export const RIGHT_CLICK_MENU_SHORTCUT_LABELS = {
     'The keyboard keys "Control" and "V"',
     'web-stories'
   ),
+  OPTION_COMMAND_C: _x(
+    'Option Command C',
+    'The keyboard keys "Option", "Command", and "C"',
+    'web-stories'
+  ),
+  OPTION_CONTROL_C: _x(
+    'Option Control C',
+    'The keyboard keys "Option", "Control", and "C"',
+    'web-stories'
+  ),
+  OPTION_COMMAND_V: _x(
+    'Option Command V',
+    'The keyboard keys "Option", "Command", and "V"',
+    'web-stories'
+  ),
+  OPTION_CONTROL_V: _x(
+    'Option Control V',
+    'The keyboard keys "Option", "Control", and "V"',
+    'web-stories'
+  ),
```

Comment:
These words are macOS-specific. What about Windows/Linux?

Why don't we say e.g. "Copy" followed by the keyboard shortcut symbols?

Reply:
line 68 and line 63 separate mac vs windows/linux and then are set like so: 
```
{
        label: RIGHT_CLICK_MENU_LABELS.COPY_IMAGE_STYLES,
        shortcut: {
          display: isMacOs ? '⌥ ⌘ C' : '⌥ ctrl C',
          title: isMacOs
            ? RIGHT_CLICK_MENU_SHORTCUT_LABELS.OPTION_COMMAND_C
            : RIGHT_CLICK_MENU_SHORTCUT_LABELS.OPTION_CONTROL_C,
        },
      },
````

Is this what you're asking about?

Reply:
I updated this in #8382, would this be better?

https://github.com/google/web-stories-wp/pull/8382/files#diff-d448253e3cf32f60ad1f345c3bbf87ff388746af5a81e6c1d2b77b758239948fR52-R96

Reply:
> These words are macOS-specific. What about Windows/Linux?
> 
> Why don't we say e.g. "Copy" followed by the keyboard shortcut symbols?

Yeah we do but this is just for the shortcut's aria label. However, we may just remove this in favor of using `kbd` elements for the shortcut labels: 
https://github.com/google/web-stories-wp/pull/8382/files#diff-d448253e3cf32f60ad1f345c3bbf87ff388746af5a81e6c1d2b77b758239948fR52-R96